requires-testing.txt

```diff
@@ -1,13 +1,12 @@
-pytest==6.0.1;python_version>="3.0"
-pytest==4.6.9;python_version=="2.7"
-pytest-sugar==0.9.4
-pytest-mock==3.2.0;python_version>="3.0"
-pytest-mock==2.0.0;python_version=="2.7"
-lxml==4.6.1
-selenium==3.141.0
-percy==2.0.2
-cryptography==3.0
-requests[security]==2.21.0
-beautifulsoup4==4.8.2
-waitress==1.4.3
-flask-talisman==0.7.0
+pytest>=6.0.2;python_version>="3.0"
+pytest>=4.6,<5;python_version=="2.7"
+pytest-sugar>=0.9.4
+pytest-mock>=3.2.0;python_version>="3.0"
+pytest-mock>=2.0.0,<3;python_version=="2.7"
+lxml>=4.6.2
+selenium>=3.141.0
+percy>=2.0.2
+requests[security]>=2.21.0
+beautifulsoup4>=4.8.2,<=4.9.3;python_version=="2.7"
```

Comment:
`beautifulsoup4` has [stated](https://pypi.org/project/beautifulsoup4/) it's about to drop Py2 support, so I set a preemptive upper bound for Py2 at the current version.